homeassistant/components/media_player/anthemav.py

```diff
@@ -0,0 +1,179 @@
+"""
+Support for Anthem Network Receivers and Processors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/media_player.anthemav/
+"""
+import logging
+import asyncio
+
+import voluptuous as vol
+
+from homeassistant.components.media_player import (
+    PLATFORM_SCHEMA, SUPPORT_TURN_OFF, SUPPORT_TURN_ON, SUPPORT_SELECT_SOURCE,
+    SUPPORT_VOLUME_MUTE, SUPPORT_VOLUME_SET, MediaPlayerDevice)
+from homeassistant.const import (
+    CONF_NAME, CONF_HOST, CONF_PORT, STATE_OFF, STATE_ON, STATE_UNKNOWN,
+    EVENT_HOMEASSISTANT_STOP)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['anthemav==1.1.5']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'anthemav'
+
+DEFAULT_NAME = "Anthem AVR"
+DEFAULT_PORT = 14999
+
+SUPPORT_ANTHEMAV = SUPPORT_VOLUME_SET | SUPPORT_VOLUME_MUTE | \
+    SUPPORT_TURN_ON | SUPPORT_TURN_OFF | SUPPORT_SELECT_SOURCE
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_HOST): cv.string,
+    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
```

Comment:
This shouldn't be necessary. If the user wants to change the name on the frontend, `customize` can be used. https://home-assistant.io/getting-started/customizing-devices/